### PR TITLE
Subtitles in Examples

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/cnx-recipes/issues/2660

### CODE REVIEW

**Review by Stackblocks — APPROVED** [review]

File: recipes/books/statistics/_config.scss [hunk 1]
```diff
@@ -188,6 +189,14 @@ $Config_FirstElements: (
   (selector: ' > .os-figure:first-child'),
 );
 
+$Config_PartType_Example:  (
```
Comment (brittany-johnson) [inline recipes/books/statistics/_config.scss:192]: This is a very random comment that I'm leaving but we should really reformat all of our maps in the config to be written like this 😂 So much easier to read I think

File: recipes/mixins/_number.scss [hunk 2]
```diff
@@ -127,12 +127,30 @@
       &:deferred {
         content: pending(bExampleHeader) content();
       }
-      [data-type="solution"] {
-        @include utils_title($solutionTitleContent, null, div, "solution-title");
+      @if $solutionNumberedTitleContent {
+        [data-type="solution"]:not(.unnumbered) {
+          @include utils_title($solutionNumberedTitleContent, null, div, "solution-title");
+        }
+        [data-type="solution"].unnumbered {
```
Comment (brittany-johnson) [inline recipes/mixins/_number.scss:134]: Can line 134 and line 138 be reduced to one selector? Just curious!

Comment (brittany-johnson) [reply recipes/mixins/_number.scss:134]: Same question for the other instances of `$solutionNumberedTitleContent` below

**Review by brittany-johnson — APPROVED** [review]
I just had one question. Looks good though!

File: recipes/mixins/_number.scss [hunk 3]
```diff
@@ -127,12 +129,30 @@
...
+  
+        [data-type="solution"]:not(.unnumbered) {
+          @include utils_title($solutionNumberedTitleContent, null, div, "solution-title");
+        }
+      } @else {
+        $mainSolutionSelector: '[data-type="solution"]';
+      }
+      #{$mainSolutionSelector} {
         @include utils_title($solutionTitleContent, null, div, "solution-title");
       }
```
Comment (wilkus27) [inline recipes/mixins/_number.scss:143]: @brittany-johnson I applied something like that. Please, let me know if this is what you meant :slightly_smiling_face:

Comment (brittany-johnson) [reply recipes/mixins/_number.scss:143]: This looks good!

**Review by brittany-johnson — APPROVED** [review]

**Comment by wilkus27** [comment]
#### Recipe changes

- modify commentary of example to use title created in the content
- add numbering to example solutions in Statistics

**Pull request merged**